website/client/components/challenges/findChallenges.vue

```diff
@@ -15,7 +15,7 @@
           .svg-icon.positive-icon(v-html="icons.positiveIcon")
           span(v-once) {{$t('createChallenge')}}
     .row
-      .col-12.col-md-6(v-for='challenge in filteredChallenges', v-if='!memberOf(challenge)')
+      .col-12.col-md-6(v-for='challenge in filteredChallenges', v-if='!memberOf(challenge) && (challenge.flagCount < 2 || user.contributor.admin)')
```

Comment:
This needs to also allow the challenge to be seen if the user is the challenge's owner (that user might not be a member of the challenge).